src/map/skill.c

```diff
@@ -2089,7 +2089,7 @@ int skill_blown(struct block_list* src, struct block_list* target, int count, in
 		case BL_MOB:
 		{
 			const struct mob_data *md = BL_UCCAST(BL_MOB, target);
-			if (md->class_ == MOBID_EMPELIUM)
+			if (md->class_ == MOBID_EMPELIUM || md->class_ >= MOBID_DUMMY_10 && md->class_ <= MOBID_DUMMY_10_FIRE)
```

Comment:
about this ... why not set in mob_db.conf ?? i remember there was a settings that disable monsters from knocked back by players.